config.json

```
@@ -16,6 +16,20 @@
   },
   "solution_pattern": "example.f90",
   "test_pattern": ".*\\_test.f90",
+  "files": {
+    "solution": [
+      "path/to/%{kebab_slug}.ext"
+    ],
+    "test": [
+      "path/to/%{kebab_slug}.ext"
+    ],
+    "example": [
+      "path/to/%{kebab_slug}.ext"
```

```suggestion
      "example.f90"
```

(these should probably be moved to `.meta/`?)